droidconKE/chai
# drop dckeswag

---
name: Pull request
about: Create a pull request
label: 'triage me'
Fixes [Referenced Issue #](https://github.com/droidconKE/chai/issues)_.


---
## How can this PR be tested?
Run the branch. dckeswag will not be present. 
dckeswag has been moved to [dckeswag project](https://github.com/droidconKE/dckeswag)

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:


Is this your first Pull Request?

Fixes #7 🦕

![]()
